tests/foreman/api/test_classparameters.py

```diff
@@ -542,14 +818,20 @@ def test_negative_validate_matcher_non_existing_attribute(self):
         2. Create a matcher with non existing attribute in org.
 
         @assert: Error raised for non existing attribute.
-
-        @caseautomation: notautomated
-
-        @CaseLevel: System
         """
+        sc_param = self.sc_params_list.pop()
+        with self.assertRaises(HTTPError) as context:
+            entities.OverrideValue(
+                smart_class_parameter=sc_param,
+                match='non_existing_attribute=example.com',
```

Comment:
Yes, here we have implemented another scenario to verify that invalid matcher cannot be created.
```
 match='fqdn=nonexisting.com' or 'hostgroup=nonexistingHG'
```
will work just perfect from CLI and I think will also work from API, so if you can provide any example from API perspective, appreciate that